sonnet/src/mixed_precision.py

```diff
@@ -77,21 +77,23 @@ def _cast_call(f, new_dtype, args, kwargs):
 def modes(valid_types):
   """Decorate a function to cast inputs/outputs to different precision.
 
-      snt.Linear.__call__ = snt.mixed_precision.modes(
-        [tf.float32, tf.float16])(snt.Linear.__call__)
-      mod = snt.Linear(10)
-      snt.mixed_precision.enable(tf.float16)
-      y = mod(tf.ones([1, 1]))  # First call will be done in F32.
-      y = mod(tf.ones([1, 1]))  # MatMul/Add will be done in F16.
+      .. code-block:: python
```

Comment:
Would you mind using `::` instead as in e.g. 

https://github.com/deepmind/sonnet/blob/3732f195416d8d0e212835beb65303abfab8929f/sonnet/src/recurrent.py#L226

Reply:
Resolved. 623a88d